code-challenges.js

```diff
@@ -13,15 +13,35 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
+//create a function that takes a num as an argument
+//decide if the number is evenly devisivible by three or not.
 
+const divisibleBy3 = (num) => {
+  if(Math.abs(num) % 3 == 0) {
```

Comment:
great work! i'm not sure if `Math.abs` is necessary in this code but i really like your approach!